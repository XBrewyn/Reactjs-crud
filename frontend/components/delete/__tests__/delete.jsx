import React from 'react';
import { create } from 'react-test-renderer';
import Homepage from '../delete';

describe('<Delete />', () => {
  it('Renders correctly', () => {
    const component = create(<Homepage />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
