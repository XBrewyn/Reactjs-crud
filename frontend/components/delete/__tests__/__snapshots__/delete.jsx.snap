// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Delete /> Renders correctly 1`] = `
<div
  className="css-m4qxtz"
>
  <h1>
    Delete
  </h1>
  <form>
    <input
      name="id"
      onChange={[Function]}
      placeholder="id"
      type="text"
      value=""
    />
    <input
      onClick={[Function]}
      type="button"
      value="delete"
    />
  </form>
</div>
`;
